index.html

```diff
@@ -53,7 +53,10 @@ <h2>
           battery. It allows data encryption and is supported by popular home
           automation platforms, like
           <a href="https://www.home-assistant.io">Home&nbsp;Assistant</a>, out
-          of the box.
+          of the box. This webpage is providing all the information about
+          <strong>the new BTHome V2 format.</strong> Information about the old
+          BTHome V1 format can be found
+          <a href="https://www.bthome.io/bthome_v1.html">here</a>.
```

Comment:
Let's move the V1 version link to the footer. 

Let's not call v2 "new".

Reply:
Ok, moved it to the footer